beacon-chain/slasher/receive.go

```diff
@@ -78,3 +105,23 @@ func (s *Service) processQueuedAttestations(ctx context.Context, epochTicker <-c
 		}
 	}
 }
+
+// Process queued blocks every time an epoch ticker fires. We retrieve
```

Comment:
Keep in mind this has liveness concern to always wait one epoch. One can say that prysm is less performant at broadcasting or holding/proposing proposer slashing.

Major exchanges/whales may not like this

Reply:
Indeed, per slot seems like is best but we want to better benchmark slasher's performance first. We will explore changing these values in future PRs that look at performance